ex03.exs

```diff
@@ -51,11 +51,33 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    # your code goes here
-    # I'm hoping to see a simple pipeline as the body of this function...
+	chunkCollection(collection, process_count) |> mapCollection(function)
   end
 
-  # and here...
+  #chunks the collection passed to pmap 
+  def chunkCollection( collection, process_count ) do
+	collection |> Enum.chunk_every( trunc(Enum.count(collection) / process_count ) )
```

Comment:
The `trunc` returns the incorrect value if the collection size is not a multiple of the process count (try 10 and 4).

# -2